Source/Scene/ClippingPlaneCollection.js

```diff
@@ -90,8 +90,7 @@ define([
             }
         }
 
-        this._enabled = false;
-        this.enabled = defaultValue(options.enabled, true); // set using setter
+        this._enabled = defaultValue(options.enabled, true);
```

Comment:
I noticed that the default value of the setter is documented as `false` but it should be `true`.